cmd/clusterctl/client/cluster/mover.go

```diff
@@ -394,7 +394,7 @@ func (o *objectMover) ensureNamespace(toProxy Proxy, namespace string) error {
 		Name: namespace,
 	}
 
-	if err := cs.Get(ctx, key, ns); err == nil {
+	if err = cs.Get(ctx, key, ns); err == nil {
```

Comment:
good catch

Reply:
```suggestion
	err = cs.Get(ctx, key, ns)
	if err == nil {
```

It's a little strange to see it the other way, given that usually the variable is shadowed

Reply:
I've been bitten by this like 5 times already from copy and pasting this line and using it elsewhere.